bindings/nodejs/examples/how_tos/client/get-outputs.ts

```diff
@@ -30,10 +30,11 @@ async function run() {
             { hasTimelock: false },
             { hasStorageDepositReturn: false },
         ]);
-        console.log('Output ids: ', outputIdsResponse, '\n');
+        console.log!('First output of query:');
```

Comment:
What's that `log!` ?`Forbidden non-null assertion`

Reply:
I have no clue how that got there 😅 
```suggestion
        console.log('First output of query:');
```